### ISSUE TITLE
[BUG] epicli upgrade fails when upgrading Kibana

### ISSUE THREAD

**Issue by to-bar** [epiphany-platform/epiphany#1635]
**Describe the bug**
There is a fatal error when upgrading Kibana:
```
TASK [kibana : Create log directory for Kibana]
FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'kibana_log_dir'\n\nThe error appears to be in '/shared/build/06todevawrhcalico/ansible/roles/kibana/tasks/setup-logging.yml': line 2, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: Create log directory for Kibana\n  ^ here\n"}
```

**To Reproduce**
1. create cluster using epicli 0.6
2. run `epicli upgrade` using a build from develop branch

**Additional context**
It looks like defaults from `core\src\epicli\data\common\ansible\playbooks\roles\kibana\defaults\main.yml` are not loaded and `specification.versions[ansible_os_family].opendistro` is loaded from `roles/opendistro_for_elasticsearch/defaults/main.yml`.

**Comment by plirglo** [comment]
Unable to test - getting error during upgrade: 
```
ERROR epicli - No such attribute: cloud
```

Similar issue what we have with task: #1647

Bug already created: #1741

**Comment by rafzei** [comment]
Fix merged into develop [#1750](https://github.com/epiphany-platform/epiphany/pull/1750)

**Comment by plirglo** [comment]
Tested on Ubuntu / Azure.

**Comment by plirglo** [comment]
Blocked on RedHat (see blockers)

**Comment by to-bar** [comment]
Upgrading Kibana also fails for v0.4.5 since there is no `opendistroforelasticsearch-kibana` package but legacy `kibana-oss`.
```
2020-10-13T17:03:00.7317205Z [38;21m17:03:00 INFO cli.engine.ansible.AnsibleCommand - TASK [upgrade : Open Distro for Elasticsearch - Kibana | Assert that opendistroforelasticsearch-kibana package is installed] ***[0m
2020-10-13T17:03:00.7671398Z [31;21m17:03:00 ERROR cli.engine.ansible.AnsibleCommand - fatal: [ec2-18-202-18-39.eu-west-1.compute.amazonaws.com]: FAILED! => {"assertion": "ansible_facts.packages['opendistroforelasticsearch-kibana'] is defined", "changed": false, "evaluated_to": false, "msg": "opendistroforelasticsearch-kibana package not found, nothing to upgrade"}[0m
```

**Comment by plirglo** [comment]
Tested on RedHat / Azure - OK

Before upgrade:
![image.png](https://images.zenhubusercontent.com/5ef9f41844c19c2f7f1c83e8/0f819bbb-033b-453e-ba51-eaf1de06d300)

After upgrade:
![image.png](https://images.zenhubusercontent.com/5ef9f41844c19c2f7f1c83e8/55cd0704-af8d-4102-b5f5-39ec05522feb)
